tests/data/hierarchy_input.info

```diff
@@ -30,3 +30,17 @@ laplace
   reordering None
   n_refinements 5
 }
+
+;; Uncomment below for running ML
+;"max levels" 1
+;coarse
+;{
+;  type ML
+;  params
+;  {
+;    "ML output" 43
+;    "max levels" 3
```

Comment:
@aprokop Shouldn't this be 2 instead of 3?

Reply:
This specifies the number of levels in ML hierarchy. I don't think it should be specified at all, allowing ML to determine it on its own.

Reply:
Will remove